routes.go

```diff
@@ -53,8 +61,23 @@ func provideCoreEndpoints() fx.Option {
 func provideCoreOption(server string, in RoutesIn) arrangehttp.Option[http.Server] {
 	return arrangehttp.AsOption[http.Server](
 		func(s *http.Server) {
+			urlPrefix := fmt.Sprintf("/%s", apiBase)
+			if in.PreviousVersionSupport {
+				urlPrefix = fmt.Sprintf("/%s", apiBaseDualVersion)
+			}
 			mux := chi.NewMux()
+			// TODO: should probably customize things a bit
+			mux.Use(recovery.Middleware(recovery.WithStatusCode(555)))
```

Comment:
is this a later todo?

Reply:
this was copied and pasted over from old code - including the comment.